experimental/README.md

```diff
@@ -32,6 +33,13 @@ true
 
 ## Current experimental features
```

Comment:
Other experimental features;

- "top level" `docker deploy` command (only the top level command, `docker stack deploy` is not experimental)
- bundle / `.dab` files for `docker deploy` / `docker stack deploy` are still experimental